e2e_tests/branch_test.go

```diff
@@ -0,0 +1,51 @@
+package e2e_tests
+
+import (
+	"testing"
+
+	"github.com/aviator-co/av/internal/git/gittest"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBranch(t *testing.T) {
```

Comment:
I'm not sure using the `av stack tree` command is a good way to test. What do you think? Are there any other good ways??